README.md

```diff
@@ -1,63 +1,57 @@
 # Talos
-A multi level authorization project
+A file system permission project.
```

Comment:
The multiple levels look like a file system, and a file system is one of the applications of this project, but there are a couple of other applications as well. Therefore, is it wise to explicitly call it a "file system permission project"?

Reply:
The most important element of this project justifying, at least to me, the description "file system permission" is the hierarchic nature of the resources.